client-rpc/src/rpc/wallet_rpc.rs

```diff
@@ -25,6 +25,12 @@ pub trait WalletRpc: Send + Sync {
     #[rpc(name = "wallet_create")]
     fn create(&self, request: WalletRequest) -> Result<String>;
 
+    #[rpc(name = "wallet_create_hd")]
+    fn create_hd(&self, request: WalletRequest) -> Result<String>;
```

Comment:
Can we update current `wallet_create` RPC call to take `WalletType` as an additional parameter?

Reply:
ok